### ISSUE TITLE
Support multiple content types

### ISSUE THREAD

**Issue by nmcb** [zalando/play-swagger#6]
Currently only application/json is provided for, we should come up with a design that makes content types pluggable and implement provision of multiple content types in order of specified preference.

**Comment by slavaschmidt** [comment]
It's pretty strait forward to add content negotiation to the controller. The problem is to bring custom defined body parsers into the scope. 

So basically, the issue is as following:
- Content negotiation is something which should happen behind scenes, therefore it should be done in controller_base, not the controller itself
- The controller_base completely regenerated each time specification is changed. There is no hand-written code in the controller_base
- For content negotiation to work we might need to have custom (de)serialisers in scope
- We cannot place them into the package object because it's already used to place PathBinders, QueryBinders and HeaderBinders and also completely generated
- So basically it seems like our only option is to have a naming convention for the programmer to place serialisers into some package/object/class/trait/whatever 
- We then need to analyse which custom serialisers present and generate dynamic imports in the controller_base accordingly, which gives us following questions to answer:
* where to place sources for custom serialisers
* which infrastructure provide for the programmer to write them
* which kind of structure they should be placed into
* how to check which serialisers to include where and avoid naming clashes

Would like to discuss this more deeply

**Comment by slavaschmidt** [comment]
Released in 0.1.6
